http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/ProxyRoutes.scala

```diff
@@ -237,17 +256,38 @@ class ProxyRoutes(proxyService: ProxyService, corsSupport: CorsSupport, refererC
     for {
       ctx <- ev.ask[AppContext]
       apiCall = proxyService.proxyRequest(userInfo, googleProject, runtimeName, request)
-      _ <- metrics.incrementCounter("proxyRuntime")
       resp <- ctx.span.fold(apiCall)(span => spanResource[IO](span, "proxyRuntime").use(_ => apiCall))
+      tool = if (request.uri.toString.contains("jupyter")) {
```

Comment:
I think we may have an ADT for this .. `RuntimeContainerServiceType`. 

I wonder if this would work (I haven't compiled it :)):
```
tool = RuntimeContainerServiceType.values.find(s => request.uri.toString.contains(s.proxySegment)).map(_.imageType.name).getOrElse("other")
```

I don't feel too strongly, but seems nice to not duplicate the proxy segment / tool name strings.

Reply:
oh yes that's much better thanks